$NetBSD: patch-include_config.h.in,v 1.1 2018/01/26 12:45:27 he Exp $

Provide a sneaky way to avoid problems resulting from the SIZE macro
expanding to two arguments, but the argument list already having been
scanned, leading to build failure with SSP on NetBSD 8.0.
Thanks for hint from joerg@

--- include/config.h.in.orig	2018-01-26 10:57:53.228900262 +0000
+++ include/config.h.in
@@ -287,13 +287,13 @@
 #endif
 
 #ifdef HAVE_SNPRINTF
-#define SNPRINTF snprintf
+#define SNPRINTF(str, ...) snprintf(str, __VA_ARGS__)
 #else
 #define SNPRINTF sprintf
 #endif
 
 #ifdef HAVE_VSNPRINTF
-#define VSNPRINTF vsnprintf
+#define VSNPRINTF(str, ...) vsnprintf(str, __VA_ARGS__)
 #else
 #define VSNPRINTF vsprintf
 #endif
